fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -1599,6 +1599,47 @@ private void transferToMaster() {
                 // Set initial root password if master FE first time launch.
                 auth.setInitialRootPassword(Config.initial_root_password);
             } else {
+                int currentVariableVersion = VariableMgr.newSessionVariable().variableVersion;
```

Comment:
How about create a new class and move these logic to there.
The Env.java is too big

Reply:
done